draft-ietf-ppm-dap.md

```diff
@@ -1236,39 +1246,73 @@ This message consists of:
   `time_interval`, the batch is not determined until the Collector's query is
   issued (see {{query}}).
 
-* `report_shares`: The sequence of report shares to aggregate. The
-  `encrypted_input_share` field of the report share is the `HpkeCiphertext`
-  whose index in `Report.encrypted_input_shares` is equal to the index of the
-  aggregator in the task's `aggregator_endpoints` to which the
-  `AggregationJobInitReq` is being sent.
+* `report_inits`: The Helper's share of information about each report to be
+  aggregated in this aggregation job. The value of the
+  `report_share.encrypted_input_share` field is taken from the
+  `Report.helper_encrypted_input_share` field in the original report.
+  `leader_prep_share` is the Leader's prepare message-share as computed during
+  the VDAF initialization process.
 
-Let `{aggregator}` denote the Helper's API endpoint. The Leader sends a PUT
-request to `{aggregator}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}`
-with its `AggregationJobInitReq` message as the payload. The media type is
+Let `{helper}` denote the Helper's API endpoint. The Leader sends a PUT request
+to `{helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}` with its
+`AggregationJobInitReq` message as the payload. The media type is
 "application/dap-aggregation-job-init-req". The Leader's aggregation job is now
 in round 0.
 
 The Helper's response will be an `AggregationJobResp` message (see
 {{aggregation-helper-init}}, which the Leader validates according to the
-criteria in {{aggregation-job-validation}}. If the message is valid, the Leader
-moves to the aggregation job continuation phase with the enclosed prepare steps,
-as described in {{agg-continue-flow}}. Otherwise, the Leader should abandon the
-aggregation job entirely.
+criteria in {{aggregation-job-validation}}. If the message is invalid, the
+Leader should abandon the aggregation job entirely.
+
```

Comment:
I think I'd like to leave this as-is -- the obvious fix that comes to mind would be to break leader/helper initialization into three sections: "leader initialization up to sending the message", "helper initialization", "leader initialization after receiving the response". I think this would be less understandable than the current forward-reference.